download/downloadSpec.go

```diff
@@ -141,6 +141,9 @@ func detectLinuxId() (string, error) {
 	}
 
 	id := osRelease["ID"]
+	if id == "manjaro" {
+		return "manjaro", nil
+	}
```

Comment:
Arch/Manjaro does not have a `VERSION_ID` property on `/etc/os-release` file, so there is not point in continuing

Reply:
@rafahop I wonder if we should support other Linux versions here... 
```suggestion
	if osName == "linux" & !strings.Contains("debian ubuntu", id) {
		return id, nil
	}
```

Reply:
Other linux versions might be using the os-release file? We'll need to identify the version in most cases to download the appropriate tar file.
When/if there is a need for extending it to other linux versions we could change this appropriately.